.github/workflows/build-mudlet.yml

```diff
@@ -435,3 +435,29 @@ jobs:
           echo "CMakeOutput.log: "
           cat "${{runner.workspace}}/b/ninja/CMakeFiles/CMakeOutput.log"
         }
+
+    - name: Create PR to update development
+      if: github.event_name == 'push' && (github.event_name == 'developement' || startsWith(github.ref, 'refs/tags/Mudlet-'))
+      run: |
+        git checkout development
+        git pull origin development
+        git checkout -b update-development-${{ github.run_number }}
+
+        APP_VERSION=$(grep 'set(APP_VERSION' CMakeLists.txt | grep -oP '\d+\.\d+\.\d+')
+        echo "Extracted APP_VERSION: $APP_VERSION"
+        echo "APP_VERSION=$APP_VERSION" >> $GITHUB_ENV
+
+        VERSION=$(grep 'VERSION =' src/mudlet.pro | grep -oP '\d+\.\d+\.\d+')
+        echo "Extracted VERSION: $VERSION"
+        echo "VERSION=$VERSION" >> $GITHUB_ENV
+
+        echo "set(APP_VERSION ${{ env.APP_VERSION }})" >> CMakeLists.txt
+        echo "VERSION = ${{ env.VERSION }}" >> src/mudlet.pro
+
+        git add CMakeLists.txt src/mudlet.pro
```

Comment:
I could be missing something, but does extracting these values and writing them back actually achieve something?

Reply:
The extraction and writing back of values were intended for a scenario where you needed to dynamically update version numbers in your files based on the extracted values.
Dont i need them?

Reply:
cc @vadi2

Reply:
I don't follow. Mind rephrasing?

Reply:
@vadi2 so i should extract `version` and `app_version` from the release event right?